statistics/statistics.js

```diff
@@ -0,0 +1,25 @@
+/* Definition of API */
+const api ="https://api.ipgeolocation.io/ipgeo?apiKey=cf2be30a619a447897245a7857d48018"; // API location
+const api2 ="https://api.covid19api.com/countries"; // API countries
+/* Definition of DOM TREE */ 
+const countries= document.querySelector("#countries"); //The input of data list
+/* Get data from API */
+async function getData(){
+var data, data2;
+    try{ const response = await fetch(api);// Get data from API location
+        const response2 = await fetch(api2); // Get data from API country   
```

Comment:
Don't put your fetchs in series in a try catch
So when the API 1 fails the same as API 2 fails
while your application can run without the first api in the search mode